Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.9.0-alpine
+MAINTAINER Lukasz Gornicki <derberg@wp.pl>
+# add bash and git
+RUN apk update && apk add --no-cache bash git openssh && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /var/cache/apk/*
+# clone and install skeleton
+RUN git clone https://github.com/YaaS/docpad-skeleton-apidocs.git
```

Comment:
yes, I know about what you wrote in last sentence.
regarding COPY, very good suggestion, I change it and instructions too, I'm also adding nano installation so nobody has to do it explicitly on the container